plugins/outputs/sensu/sensu_test.go

```diff
@@ -0,0 +1,69 @@
+package sensu
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetEndpoint(t *testing.T) {
+	ts := httptest.NewServer(http.NotFoundHandler())
+	defer ts.Close()
+
+	_, err := url.Parse(fmt.Sprintf("http://%s", ts.Listener.Addr().String()))
+	require.NoError(t, err)
+
+	t.Run("default-user-agent", func(t *testing.T) {
+		var err = error(nil)
+		require.NoError(t, err)
+	})
+}
```

Comment:
What is this test for? Are you testing if the `httptest` server can be set-up? If this fails we will be in big trouble anyway I think.

Reply:
Removed this essentially useless test and added sufficient testing for connect and write, increased coverage from 12% to 70%.

Reply:
Good job! Please consider this resolved.